ui/src/components/__tests__/TableMenu.test.tsx

```diff
@@ -310,7 +310,8 @@ describe("TableMenu component", () => {
 		);
 	});
 
-	it("CSV export fails", async () => {
+	//FIXME: Fails non-deterministically.
```

Comment:
is this something for a future PR?

Reply:
Yep, I checked with Dave and he agreed that we should mark it as skipped for now and address it down the line later (he has some ideas about what might be going wrong).